src/c++/perf_analyzer/genai-perf/genai_perf/llm_metrics.py

```diff
@@ -566,7 +577,7 @@ def _preprocess_response(
                 res_timestamps.pop()
                 res_outputs.pop()
 
-    def _tokenize_request_inputs(self, req_inputs: dict) -> list[list[int]]:
+    def _tokenize_request_inputs(self, req_inputs: dict) -> BatchEncoding:
```

Comment:
Are we actually returning `BatchEncoding` typed object? I thought we were returning a list object.

Reply:
Thanks, fixed. This one took me a while. I went deeper into the documentation, and I think it should actually be a list of ints: https://huggingface.co/docs/transformers/en/glossary#input-ids